### ISSUE TITLE
c/libblake3.pc.in have incorrect libdir / includedir

### ISSUE THREAD

**Issue by vt-alt** [BLAKE3-team/BLAKE3#311]
There are some problems with `c/libblake3.pc.in`:

```
prefix=@CMAKE_INSTALL_PREFIX@
exec_prefix=@CMAKE_INSTALL_PREFIX@
libdir=@CMAKE_INSTALL_LIBDIR@
includedir=@CMAKE_INSTALL_INCLUDEDIR@

Name: @PROJECT_NAME@
Description: @PROJECT_DESCRIPTION@
Version: @PROJECT_VERSION@

Requires:
Libs: -L${libdir} -lblake3
Cflags: -I${includedir} @BLAKE3_PKGCONFIG_CFLAGS@
```
On x86_64 system, this adds `-Llib64` and `-Iinclude` which are incorrect values (non-absolute paths thus leading to non-system dirs).
```
builder@x86_64:/.in$ pkg-config libblake3 --libs
-Llib64 -lblake3
builder@x86_64:/.in$ pkg-config libblake3 --cflags
-DBLAKE3_DLL -Iinclude
```
`libdir=` should be defined as `/usr/lib64` and `include=` as `/usr/include`.

Perhaps (based on other .pc files I have seen) you should change it to something like:

```
libdir=${exec_prefix}/@CMAKE_INSTALL_LIBDIR@
includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
```
With this change output is correct:
```
builder@x86_64:/.in$ pkg-config libblake3 --libs
-lblake3
builder@x86_64:/.in$ pkg-config libblake3 --cflags
-DBLAKE3_DLL
```

ps. Also `c/libblake3.pc.in` have no newline at the end of file.

**Comment by oconnor663** [comment]
@BurningEnlightenment I'm going to keep tagging you on CMake reports, but if you find that annoying please let me know :) In any case, no pressure to respond quickly.

**Comment by oconnor663** [comment]
Also @SteveGremory.

**Comment by oconnor663** [comment]
Auto-closed because the fix was merged, but @vt-alt can you confirm the fix?

**Comment by vt-alt** [comment]
I did not sure  pkg-config can understand `-L"${libdir}"` with quotes (since nobody AFAIK uses it like this) but it's worked OK.

**Comment by vt-alt** [comment]
BTW
```
Version: 1.3.3
```

**Comment by oconnor663** [comment]
Ah thanks for pointing that out. I've fixed the version in 74220e2ca157e1319f5141bbd2bcdfb2bc1a20a0 and updated my release instructions so that I don't forget this file next time.
